lib/admin/components/ServerSettings.js

```diff
@@ -0,0 +1,403 @@
+                    onClick={this._onRefresh}>
+                    <Glyphicon glyph='refresh' /> {this.messages('deployment.otpServers.refresh')}
+                  </Button>
+                  <Button
+                    bsStyle='primary'
+                    data-test-id='add-server-button'
+                    onClick={this._onAddServer}>
+                    <Glyphicon glyph='plus' /> {this.messages('deployment.otpServers.new')}
+                  </Button>
+                </ButtonToolbar>
+                <h4 style={{margin: 0, padding: 0, paddingTop: '10px'}}>
+                  <Icon type='server' /> {this.messages('deployment.otpServers.title')}
+                </h4>
+              </Col>
+            </Row>
+          }>
+          <div>
+            {otpServers && otpServers.length
+              ? otpServers.map((server, index) => (
+                <CollapsiblePanel
+                  key={`server-${index}`}
+                  index={index}
+                  fields={SERVER_FIELDS}
+                  defaultExpanded={!server.name}
+                  title={server.name
+                    ? <span>
+                      {server.name}{'  '}
+                      <small>{server.publicUrl}</small>
+                    </span>
+                    : `[${this.messages('deployment.otpServers.serverPlaceholder')}]`
+                  }
+                  data={server}
+                  saveDisabled={!this.state.hasEdits}
+                  onRemove={this._onDeleteServer}
+                  onSave={this._onSaveServer}
+                  onChange={this._getOnChange}>
+                  <ServerSpecialFields
+                    index={index}
+                    getOnChange={this._getOnChange}
+                    server={server}
+                    terminateEC2Instances={terminateEC2Instances}
+                    onChangeProject={this._onChangeProject}
+                    onToggleEc2Info={this._onToggleEc2Info}
+                    projects={projects} />
+                </CollapsiblePanel>
+              ))
+              : <p className='lead text-center'>No servers defined</p>
+            }
+          </div>
+        </Panel>
+        <Panel>
+          <div>
+            <h3>Instructions</h3>
+            <p>
+              Here you can create and manage servers/targets to deploy bundled transit and
+              street network data to. There are a handful of ways to configure
+              these servers, but the below instructions detail the generally
+              recommended approaches.
+            </p>
+            <h4>Production deployments</h4>
+            <p>
+              The recommended approach for production deployments is to create
+              an elastic load balancer (ELB) in AWS along with a target group.
+              You should then reference the target group ARN in the server
+              (an S3 bucket must be identified as well). The steps for this
+              approach are as follows:
```

Comment:
There could definitely be more documentation for this process and how to fill in the fields appropriately. Better yet, maybe there could be some kind of server-side logic that would automatically setup a load balancer.

Reply:
Could you suggest some changes to the documentation, since you went through the process of setting it up somewhat blind?

Regarding the auto-setup by the server, I want to be careful not to over-automate this. We likely need this to be flexible enough to work with across AWS accounts/networks and I'm not sure if there is a great way to account for that with an automatic procedure. For example, if we have Transit Agency X that wants IBI prod Data Tools to deploy to an ELB in their own AWS account, we can set up the IAM permissions accordingly, but there may be too many factors to handle an auto-setup.

Reply:
Making these docs into a step-by-step guide with screenshots in the documentation is what I would've liked to see. And regarding the automatic ELB setup, I was thinking it could be a good default, but that option would require more maintenance and support.